package.json

```diff
@@ -24,7 +24,7 @@
   "author": "YetiForceCompany <info@yetiforce.com> (https://yetiforce.com)",
   "license": "YetiForce Public License 6.5",
   "dependencies": {
-    "@fortawesome/fontawesome-free": "^6.1",
```

Comment:
Updated @fortawesome/fontawesome-free to version ^6.6.0. Verified compatibility with the project and tested the changes locally. Everything works as expected. Ready to merge.